### DISCUSSION TITLE
为什么特意排除了 15 和 17 两个 node 版本

### DISCUSSION THREAD

**Discussion by ninesunsabiu** [umijs/father#752] [Q&A] [answered]
刚好有同事用了 node 17 然后构建失败，排查到这里，发现是特意排除了这两个版本  
但是当时的 commit 并没有关联其他的 issue，所以特别想知道原因
是这两个版本的 Node 有什么问题吗

**Comment by PeachScript** [comment, marked as answer]
奇数版本的 Node.js 是给生态开发者验证用的，不会进 LTS

**Reply by ninesunsabiu** [reply]
我正在想是不是这个原因，因为看到 eslint plugin 的脚手架生成也是 `"node": "^14.17.0 || ^16.0.0 || >= 18.0.0"` 

感谢
